recipes/d3/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "d3" %}
+{% set version = "4.10.2" %}
+{% set sha256 = "eaa296738d4ec6bc49ebfe6635284972b38db5f146ba9f5778af45b6306044ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/d3/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    - nodejs
+test:
+  commands:
+    - npm list -g d3
```

Comment:
Can you have a look at https://github.com/conda-forge/staged-recipes/pull/4047 and do the same here?